extensions/wikia/AssetsManager/builders/AssetsManagerSassesBuilder.class.php

```diff
@@ -21,6 +21,10 @@ public function __construct(WebRequest $request) {
 			$builder = new AssetsManagerSassBuilder($sassRequest);
 
 			$this->mContent .= "\n\n/* $file */\n" . $builder->getContent();
+
+			if (strpos($this->mOid, 'ads.scss') !== false) {
```

Comment:
Just an idea here: we're using `/sasses/` AssetsManager entry point to render and fetch several SCSS in a single request. What if we make `/sasses/` entry point support both AM groups and single files? Using this approach we can introduce `ads` AM group that will have `crossorigin` setting in AM's `config.php`' - hence causing the `/sasses/` response that contains `ads` group to have a proper HTTP header set.

#### Pros:

* you can tweak `crossorigin` setting on per-group basis
* no MW core "hacking" - adding HTML attributes etc.
* more secure as we do not rely on client-provided URL parameter that controls `crossorigin` header

#### Cons:

* can't think of any now ;)